userappwithbroker/src/main/java/com/microsoft/aad/adal/example/userappwithbroker/MainActivity.java

```diff
@@ -320,12 +328,14 @@ private void callAcquireTokenSilent(final String resource, final String userUniq
 
             @Override
             public void onSuccess(AuthenticationResult authenticationResult) {
+                verifyThread();
```

Comment:
furthermore, in the method body of "showMessage", could we use something similar to:
        runOnUiThread(new Runnable() {
            @Override
            public void run() {
                Toast.makeText(MainActivity.this, msg, Toast.LENGTH_LONG).show();
            }
        });

thanks